reference/inertialunit.md

```diff
@@ -4,7 +4,7 @@ Derived from [Device](device.md).
 
 ```
 InertialUnit {
-  MFVec3f    lookupTable    []    # interpolation
+  MFVec3f    lookupTable    [ ]   # interpolation
   SFBool     xAxis          TRUE  # compute roll
   SFBool     zAxis          TRUE  # compute pitch
   SFBool     yAxis          TRUE  # compute yaw
```

Comment:
Note: the spacing is not consistent between the data type and the field name and between the field name and the default value and between the default value and the comment. Sometimes, we have a minimum amount of spaces (up to 1) and sometimes, we have an arbitrary number of spaces, like here 4 or 5.